source/common/upstream/upstream_impl.cc

```diff
@@ -1827,20 +1828,7 @@ getDnsLookupFamilyFromCluster(const envoy::config::cluster::v3::Cluster& cluster
 
 Network::DnsLookupFamily
 getDnsLookupFamilyFromEnum(envoy::config::cluster::v3::Cluster::DnsLookupFamily family) {
```

Comment:
wow, nice clean up!  To go even further, can we remove this entirely, and change line 1826 to call the utility?